web_src/js/features/imagediff.js

```diff
@@ -110,29 +110,42 @@ export function initImageDiff() {
     const $imagesAfter = imageInfos[0].$images;
     const $imagesBefore = imageInfos[1].$images;
 
-    initSideBySide(createContext($imagesAfter[0], $imagesBefore[0]));
+    initSideBySide(this, createContext($imagesAfter[0], $imagesBefore[0]));
     if ($imagesAfter.length > 0 && $imagesBefore.length > 0) {
       initSwipe(createContext($imagesAfter[1], $imagesBefore[1]));
       initOverlay(createContext($imagesAfter[2], $imagesBefore[2]));
     }
 
-    $container.find('> .image-diff-tabs').removeClass('is-loading');
+    for (const el of this.querySelectorAll(':scope > .image-diff-tabs')) {
+      el.classList.remove('is-loading');
+    }
```

Comment:
There should be one and only one `image-diff-tabs`, so I think it could be:

```suggestion
    this.querySelector(':scope > .image-diff-tabs').remove('is-loading');
```

Reply:
[`ecb3b1c` (#30140)](https://github.com/go-gitea/gitea/pull/30140/commits/ecb3b1c93303f20f226c1f47fad2439ba1b16aa1)